packages/shared/src/components/comments/SubComment.tsx

```diff
@@ -58,9 +61,9 @@ function SubComment({
           {...inputProps}
           className={className}
           post={props.post}
-          onCommented={(data, isNew) => {
+          onCommented={(...params) => {
             onReplyTo(null);
-            onCommented(data, isNew);
+            onCommented(...params);
```

Comment:
What's the difference between these 2?

Reply:
I mean `onReplyTo` and `onCommented`

Reply:
`onReplyTo` is some state holder for which comment they are replying to. Since it was successful, the `replyTo` state is getting null in this one.

Reply:
Ah good, wasn't sure if it was the success callback maybe :) 
In that case I have my answer